lms/envs/devstack_optimized.py

```diff
@@ -23,10 +23,7 @@
 
 ########################## Devstack settings ###################################
 
-if 'BOK_CHOY_HOSTNAME' in os.environ:
-    from .devstack_docker import *  # pylint: disable=wildcard-import, unused-wildcard-import
-else:
-    from .devstack import *  # pylint: disable=wildcard-import
+from .devstack import *  # pylint: disable=wildcard-import
```

Comment:
I couldn't understand why this check was necessary before so I simply removed the check and imported the devstack.py (which now has all the configs from devstack_docker.py)